gallery/setupTests.js

```diff
@@ -12,7 +12,12 @@ jest.retryTimes(3);
 
 const toMatchImageSnapshot = configureToMatchImageSnapshot({
   customDiffDir: path.join(__dirname, 'test-results'),
-  customSnapshotIdentifier: ({ defaultIdentifier }) => defaultIdentifier
+  // eslint-disable-next-line no-undef
+  customSnapshotsDir: process.env.RSC_GALLERY_THEME === 'DARK'
```

Comment:
Rather than doing this comparison twice, I'd say move it out to a local variable above.

Reply:
> Rather than doing this comparison twice, I'd say move it out to a local variable above.

Absolutely. Stored in variable and removed eslint warning here: [3d57fe2](https://github.com/sonatype/sonatype-react-shared-components/pull/808/commits/3d57fe24579829f1c6cfaf4ffe28d1d6838a7d68)